### PR TITLE
PEP8

### PR DESCRIPTION
**Pull request by jklenzing**
This was intended to be another documentation branch, but I started with an automated PEP8 check and found enough that we should probably look over the changes.  Not everything is updated (some errors will likely be fixed by improving #16), and not everything is necessarily good.  Lots of places where the code overflows 80 characters, but I'm not sure how to fix without rewriting large bits of code.

### CODE REVIEW

**Comment by rstoneback** [comment]
Pep8 should be considered a guideline and not a rigid set of rules. Especially on the line length. 80 chars is generally too few for science.

I am wary of automated changes. I’ll put this on my list of review tasks.


On Nov 27, 2018, at 6:59 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


This was intended to be another documentation branch, but I started with an automated PEP8 check and found enough that we should probably look over the changes. Not everything is updated (some errors will likely be fixed by improving #16<https://github.com/rstoneback/pysat/issues/16>), and not everything is necessarily good. Lots of places where the code overflows 80 characters, but I'm not sure how to fix without rewriting large bits of code.

________________________________

File: pysat/_meta.py [hunk 1]
```diff
@@ -1079,19 +1093,19 @@ def from_csv(cls, name=None, col_names=None, sep=None, **kwargs):
         elif not os.path.isfile(name):
```
Comment (rstoneback) [inline pysat/_meta.py]: spacing in block below one tab too many

Comment (jklenzing) [reply pysat/_meta.py]: fixed

**Comment by jklenzing** [comment]
No problem.  The only automated changes was whitespace removal, since Atom takes care of that for me.  Everything else is me working from an automated report.

**Comment by rstoneback** [comment]
Thanks! Looks like I got a bit ahead of myself there. I’ll get back to the review after I’m able to sort out some testing issues that I’ve introduced in my own branch. :)

Cheers.

On Nov 28, 2018, at 2:17 AM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


No problem. The only automated changes was whitespace removal, since Atom takes care of that for me. Everything else is me working from an automated report.

**Comment by jklenzing** [comment]
coltype == type() is working better than isinstance here.  Not sure why.  Reverting back for now.

**Comment by rstoneback** [comment]
I believe one of the earlier methods stores the data type itself, or something like that. It is intentional though a bit nonstandard.

Cheers.

On Nov 30, 2018, at 9:03 AM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


coltype == type() is working better than isinstance here. Not sure why. Reverting back for now.

File: pysat/_files.py [hunk 2]
```diff
@@ -269,35 +271,34 @@ def _load(self, prev_version=False):
...
                                    data_path=self.data_path,
                                    format_str=self.file_format)
 
         if not info.empty:
             print('Found {ll:d} of them.'.format(ll=len(info)))
         else:
-            estr = "Unable to find any files that match the supplied template. "
-            estr += "If you have the necessary files please check pysat "
+            estr = "Unable to find any files that match the supplied template."
+            estr += " If you have the necessary files please check pysat "
```
Comment (rstoneback) [inline pysat/_files.py]: Seems a bit weird to have the space at the front of the sentence. Did the pep8 checker suggest this?

Comment (jklenzing) [reply pysat/_files.py]: I think this was me trying to get to below 80 characters.  The checker just points out violations, doesn't suggest corrections

Comment (jklenzing) [reply pysat/_files.py]: fixed.

**Comment by jklenzing** [comment]
@rstoneback, 2.7 is passing except for 8 download-related tests.

**Comment by jklenzing** [comment]
reset to remove last commit with spacing update.  Ready to go.

File: pysat/_orbits.py [hunk 3]
```diff
@@ -103,14 +103,14 @@ def __init__(self, sat=None, index=None, kind=None, period=None):
             raise ValueError('Unknown kind of orbit requested.')
 
         self._orbit_breaks = []
-        self.num = 0 #[]
+        self.num = 0  # []
```
Comment (aburrell) [inline pysat/_orbits.py]: Shouldn't we just remove this old way of doing things?

File: pysat/model_utils.py [hunk 4]
```diff
@@ -304,13 +305,16 @@ def collect_inst_model_pairs(start=None, stop=None, tinc=None, inst=None,
...
 
             if not inst.empty and inst.index[0] >= istart:
-                added_names = extract_modelled_observations(inst=inst, \
-                                model=mdata, inst_name=inst_name,
-                                                            mod_name=mod_name, \
-                                mod_datetime_name=mod_datetime_name, \
-                                mod_time_name=mod_time_name, \
-                                mod_units=mod_units, sel_name=sel_name, \
-                                method=method, model_label=model_label)
+                added_names = extract_modelled_observations(inst=inst,
```
Comment (aburrell) [inline pysat/model_utils.py]: Doesn't this cause the lines to be longer than 80 characters?

File: pysat/model_utils.py [hunk 5]
```diff
@@ -348,11 +352,11 @@ def collect_inst_model_pairs(start=None, stop=None, tinc=None, inst=None,
                             matched_inst.data = inst.data.isel(im)
                     else:
                         if inst.pandas_format:
-                            matched_inst.data = matched_inst.data.append( \
+                            matched_inst.data = matched_inst.data.append(
```
Comment (aburrell) [inline pysat/model_utils.py]: Is it now allowable to change the indentation without specifying that is what is being done?

File: pysat/_constellation.py [hunk 6]
```diff
@@ -237,14 +238,15 @@ def add(self, bounds1, label1, bounds2, label2, bin3, label3,
         binx = None
 
         # TODO modify output
-        out_2d = _calc_2d_median(ans, data_label, binx, biny, xarr, yarr, zarr, numx, numy, numz)
+        out_2d = _calc_2d_median(ans, data_label, binx, biny, xarr, yarr,
+                zarr, numx, numy, numz)
```
[the remark this answers is not in the document]
Comment (jklenzing) [reply pysat/_constellation.py]: fixed

File: pysat/_orbits.py (hunk 3 above)
Comment (jklenzing) [reply pysat/_orbits.py]: agreed

File: pysat/model_utils.py (hunk 4 above)
Comment (jklenzing) [reply pysat/model_utils.py]: to preserve readability, I agree.  Changing back

File: pysat/model_utils.py (hunk 5 above)
Comment (jklenzing) [reply pysat/model_utils.py]: changing back

**Pull request merged**